### ISSUE TITLE
Copied content does not appear in Maccy

### ISSUE THREAD

**Issue by gal064** [p0deje/Maccy#898]
### Before Submitting Your Bug Report

- [X] I have verified that there isn't already an issue reporting the same bug to prevent duplication.
- [X] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Maccy Version (see 'About' window)

2.0.1

### macOS Version

14.2.1 (23C71)

### Maccy Settings

```Shell
{
    "KeyboardShortcuts_delete" = "{\\"carbonKeyCode\\":51,\\"carbonModifiers\\":2048}";
    "KeyboardShortcuts_pin" = "{\\"carbonKeyCode\\":35,\\"carbonModifiers\\":2048}";
    "KeyboardShortcuts_popup" = "{\\"carbonModifiers\\":768,\\"carbonKeyCode\\":8}";
    "LaunchAtLogin__hasMigrated" = 1;
    "NSStatusItem Visible Item-0" = 0;
    "NSStatusItem Visible Item-1" = 0;
    "NSWindow Frame SUStatusFrame" = "100 606 400 134 0 0 1920 1055 ";
    "NSWindow Frame SUUpdateAlert" = "2162 394 620 398 1512 -98 1920 1055 ";
    "NSWindow Frame com.sindresorhus.Preferences.FrameAutosaveName" = "739 299 542 436 0 0 1920 1055 ";
    "NSWindow Frame com.sindresorhus.Settings.FrameAutosaveName" = "501 366 510 473 0 0 1512 944 ";
    SUEnableAutomaticChecks = 1;
    SUHasLaunchedBefore = 1;
    SULastCheckTime = "2024-09-24 18:39:31 +0000";
    SUUpdateGroupIdentifier = 424171284;
    WebKitDefaultFontSize = 13;
    WebKitJavaEnabled = 0;
    WebKitJavaScriptEnabled = 0;
    WebKitPluginsEnabled = 0;
    WebKitStandardFont = "-apple-system-font";
    WebKitUserStyleSheetEnabledPreferenceKey = 1;
    WebKitUserStyleSheetLocationPreferenceKey = "/Applications/Maccy.app/Contents/Frameworks/Sparkle.framework/Resources/ReleaseNotesColorStyle.css";
    avoidTakingFocus = 1;
    clearOnQuit = 1;
    enabledPasteboardTypes =     (
        "public.png",
        "public.html",
        "public.utf8-plain-text",
        "public.tiff",
        "public.rtf"
    );
    history = {length = 42, bytes = 0x62706c69 73743030 a0080000 00000000 ... 00000000 00000009 };
    historySize = 101;
    ignoredPasteboardTypes =     (
        "com.typeit4me.clipping",
        "net.antelle.keeweb",
        "com.agilebits.onepassword",
        "de.petermaurer.TransientPasteboardType",
        "Pasteboard generator type"
    );
    imageMaxHeight = 40;
    migrations =     {
        "2024-07-01-version-2" = 1;
    };
    pasteByDefault = 1;
    pinTo = bottom;
    popupPosition = center;
    showFooter = 1;
    showInStatusBar = 0;
    showRecentCopyInMenuBar = 0;
    showSearch = 1;
    showTitle = 1;
    windowSize = "[450,800]";
}
```

### Description

Copied string do not appear in maccy and search does not work. 

### Steps to Reproduce

1. Copy something
2. Open maccy
3. Copied string does not appear
4. Searching for something does not work

**Comment by p0deje** [comment]
This is now fixed in 2.0.1 though you might have to double-check if "Text" is enabled in the "Storage" settings.

**Comment by gal-checksum** [comment]
This is in version 2.0.1 though
